v1/src/test/java/com/google/cloud/teleport/spanner/ddl/InformationSchemaScannerIT.java

```diff
@@ -156,6 +156,7 @@ public void tableWithAllTypes() throws Exception {
             + " `arr_proto_field_2`     ARRAY<`com.google.cloud.teleport.spanner.tests.Order`>,"
             + " `arr_nested_enum`       ARRAY<`com.google.cloud.teleport.spanner.tests.Order.PaymentMode`>,"
             + " `arr_enum_field`        ARRAY<`com.google.cloud.teleport.spanner.tests.TestEnum`>,"
+            + " `hidden_column`         STRING(MAX) HIDDEN,"
```

Comment:
Have you tested this DDL in spanner prod? I hope this feature is available in prod.

Reply:
The feature is not new in Spanner, I originally added support for HIDDEN with search indexes and restricted it at the time to only generated columns. This change is extending HIDDEN support to non-generated columns as well as is supported by GoogleSQL SDL. I just retested in prod and found a missing change and also added a test to cover this.